docs/vocs/docs/pages/book/advanced-usage/sdk.mdx

```diff
@@ -81,35 +113,50 @@ OpenVM supports generating three types of proofs. The sections below describe ho
 
 ### Generating App Proofs
 
-After building and transpiling a program, you can then generate a proof. To do so, you need to commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
+After building and transpiling a program, you can then generate a proof. To do so, you need to convert your ELF into a `VmExe`, commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
+
+The SDK, however, combines the first few steps for you. Create a prover and pass in your input as a `StdIn` to generate an app proof.
 
 ```rust
-// [!include ~/snippets/examples_sdk/sdk_stark.rs:proof_generation]
+// [!include ~/snippets/examples_sdk/sdk_app.rs:proof_generation]
 ```
 
 For large guest programs, the program will be proved in multiple continuation segments and the returned `proof: ContinuationVmProof` object consists of multiple STARK proofs, one for each segment.
 
+:::info
+The app prover constructor can take an `Elf`, `VmExe`, or `SharedVmExe` (i.e. `Arc<VmExe>`). This also holds for the STARK and EVM provers specified below.
```

Comment:
```suggestion
The app prover constructor can take an `Elf`, `VmExe`, or `Arc<VmExe>`. This also holds for the STARK and EVM provers specified below.
```

Also the `&[u8]` and `Vec<u8>` types for raw ELF bytes are supported (there are some `From` implementations)